### PR TITLE
Fix for chrome service failing for multi remote #7055

### PR DESCRIPTION
**Pull request by praveendvd**
This will fix the : 

https://github.com/webdriverio/webdriverio/issues/7055

### CODE REVIEW

File: src/launcher.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ const DEFAULT_CONNECTION = {
 }
 
 const isMultiremote = obj => typeof obj === 'object' && !Array.isArray(obj)
-const isChrome = cap => cap.browserName.toLowerCase() === 'chrome'
+const isChrome = cap => cap.browserName ? cap.browserName.toLowerCase() === 'chrome' : false
```
Comment (christian-bromann) [inline src/launcher.js:24]:
```suggestion
const isChrome = cap => cap.browserName && cap.browserName.toLowerCase() === 'chrome'
```

**Review by christian-bromann — COMMENTED** [review]
Can we add a test for this?

**Comment by praveendvd** [comment]
i have made the changes @christian-bromann  and tested it thanks for the suggestion :)

**Comment by christian-bromann** [comment]
Can we add a unit test for this? Or change an existing test to make sure this works?

**Comment by praveendvd** [comment]
@christian-bromann  could i add one more capability to the mutliremote in https://github.com/webdriverio-community/wdio-chromedriver-service/blob/main/tests/launcher.test.js

**Comment by christian-bromann** [comment]
sure

**Comment by praveendvd** [comment]
Add unit test , do you want me to force push this to a single commit ? or this is ok @christian-bromann

**Comment by christian-bromann** [comment]
I believe this will be fine. Thank you!

**Review by christian-bromann — APPROVED** [review]
👍

**Pull request merged**